airbyte-integrations/connectors/destination-databricks/src/test-integration/java/io/airbyte/integrations/destination/databricks/DatabricksDestinationAcceptanceTest.java

```diff
@@ -132,18 +136,15 @@ protected void tearDown(final TestDestinationEnv testEnv) throws SQLException {
 
     // clean up database
     LOGGER.info("Dropping database schema {}", databricksConfig.getDatabaseSchema());
-    try (final Database database = getDatabase(databricksConfig)) {
+    try (final DSLContext dslContext = DSLContextFactory.create(DatabricksConstants.DATABRICKS_USERNAME,
+        databricksConfig.getDatabricksPersonalAccessToken(), DatabricksConstants.DATABRICKS_DRIVER_CLASS,
+        DatabricksDestination.getDatabricksConnectionString(databricksConfig), SQLDialect.DEFAULT)) {
+      final Database database = new Database(dslContext);
       // we cannot use jooq dropSchemaIfExists method here because there is no proper dialect for
       // Databricks, and it incorrectly quotes the schema name
       database.query(ctx -> ctx.execute(String.format("DROP SCHEMA IF EXISTS %s CASCADE;", databricksConfig.getDatabaseSchema())));
     } catch (final Exception e) {
       throw new SQLException(e);
     }
   }
-
-  private static Database getDatabase(final DatabricksDestinationConfig databricksConfig) {
-    final DSLContext dslContext = DSLContextFactory.create(DatabricksConstants.DATABRICKS_USERNAME, databricksConfig.getDatabricksPersonalAccessToken(), DatabricksConstants.DATABRICKS_DRIVER_CLASS, DatabricksDestination.getDatabricksConnectionString(databricksConfig), SQLDialect.DEFAULT);
-    return new Database(dslContext);
-  }
```

Comment:
The helper method here did not seem too bad, since it was referenced twice. Also it made the `try` statement shorter.

Reply:
This has now been resolved.